cmd/ketch/builder_list_test.go

```diff
@@ -67,3 +68,52 @@ func TestBuilderList(t *testing.T) {
 		require.Equal(t, tt.expected, buff.String())
 	}
 }
+
+func TestBuilderList_Names(t *testing.T) {
+	tests := []struct {
+		name string
+
+		builderList BuilderList
+		filter      []string
+		want        []string
+	}{
+		{
+			name: "no filter, return all",
+			builderList: BuilderList{
+				{Image: "img1"},
+				{Image: "img2"},
+				{Image: "img3"},
+			},
+			filter: nil,
+			want:   []string{"img1", "img2", "img3"},
+		},
+		{
+			name: "filtered result",
```

Comment:
This test seems to be the same as the one above it. Based off the name, I think that the filter should maybe be something like "img1" instead of nil.

Reply:
good catch, modified 2 tests